### ISSUE TITLE
View multiple tables at the same time

### ISSUE THREAD

**Issue by sonicsmooth** [sqlitebrowser/sqlitebrowser#2381]
Describe the new feature
--------------------------
I think it would be very helpful to be able to view multiple tables at the same time.
This could be done with additional "browse data" tabs which can be dragged around and repositioned, although of course that would be a lot of work.  The quick-n-dirty way would use Qt's QDockWidgets, but that is not always the best implementation of dockable windows, in my opinion.  Perhaps within the Browse Data tab there could be a button that creates a new vertical draggable separator that splits the table, and within each subwindow you can select the table as normal.  This concept could extend to the "Execute SQL" tab -- if you have multiple SQL queries and wish to see their results simultaneously that would be great.

Awesome tool!

Additional info
---------------
Please answer these questions before submitting your feature request.

**Comment by MKleusberg** [comment]
Thank you for the suggestion! This is actually an excellent time to ask about this because we're currently working on this feature. There is a whole discussion going on in #2283 where we collect different ideas. The current state which implements this via QDockWidgets can be downloaded [here](https://nightlies.sqlitebrowser.org/latest/). Are you interested in giving that version a try? Maybe it's already enough for you use case. And if not, we're definitely interested in what can still be improved :smile:

**Comment by sonicsmooth** [comment]
Yes I'll give it a try.  As an aside ive ben working on my own qt classes that allow a more flexible MDI system with multiple views of the same data. On Aug 25, 2020 8:54 AM, "Martin Kleusberg notifications-at-github.com |github/Main Spam allow|" <14846il7h7uiuat@sneakemail.com> wrote:

Thank you for the suggestion! This is actually an excellent time to ask about this because we're currently working on this feature. There is a whole discussion going on in #2283 where we collect different ideas. The current state which implements this via QDockWidgets can be downloaded here. Are you interested in giving that version a try? Maybe it's already enough for you use case. And if not, we're definitely interested in what can still be improved 😄

**Comment by chrisjlocke** [comment]
You're more than welcome to contribute to our project ...

**Comment by sonicsmooth** [comment]
After all these years I’m still never confident enough in my skills to contribute anything meaningful.  I’ve got a very full time job and 2 little kids!, and an EE background not CS.  No time for anything.  Still, I’ll contribute when I can get this thing debugged and put into standalone classes rather than the very specific use case I’ve been targeting… Might be years. 😊
